suse-ignored-rpaths.conf

```diff
@@ -0,0 +1,11 @@
+/usr/lib
+/usr/lib64
+/usr/local/lib
+/usr/local/lib64
+lib
+lib64
```

Comment:
was this really meant to be a relative path?

Reply:
no, I think that's an error from the start ...
( echo %_libdir 
  echo /usr/local/%_lib
  echo %_lib
  echo %_libdir/Xaw3d
  echo /usr/x86_64-suse-linux/lib
  echo /opt/kde3/%_lib
)  > $RPM_BUILD_ROOT/etc/suse-ignored-rpaths.conf

should have been "/%_lib" ...
can you fix that and then I can merge the PR ?